src/Bounds.cpp

```diff
@@ -564,9 +564,9 @@ class Bounds : public IRVisitor {
         } else if (can_prove(b.min == b.max)) {
             Expr e1 = a.has_lower_bound() ? a.min / b.min : a.min;
             Expr e2 = a.has_upper_bound() ? a.max / b.max : a.max;
-            if (is_positive_const(b.min) || op->type.is_uint()) {
+            if (can_prove(b.min >= 0) || op->type.is_uint()) {
```

Comment:
op->type.is_uint() is cheaper, so switch the order of the args to this `||`

Reply:
switched in 03dba70